build/chip/chip_test_suite.gni

```diff
@@ -12,12 +12,28 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+enable_efr32_build_per_testsource = true  #++++ TODO: Figure out how to enable things based on `chip_device_platform == "efr32"` which is not currently in scope.
```

Comment:
Having this flag here is just a temporary solution, since I didn't yet figure out how to access the value of `chip_device_platform` in this scope in order to test if we're building for efr32.